File: components/scroll-up/scroll-up.clj
(defn main [_ _ _ _]
  {:depends-on [{:type :js :src  "/components/scroll-up/scroll-up.js"}
                {:type :css :src "/components/scroll-up/scroll-up.css"}]
   :body [:div.git-hist-table
          [:button.scroll-up-button "Scroll up"]]})

File: components/ScrollUp/ScrollUp.ts
const ScrollUp = () => ({
  dependsOn: [
    { src: "/components/ScrollUp/scroll-up.ts" },
    { src: "/components/ScrollUp/scroll-up.css" },
  ],
  body: [
    "div",
    { class: "git-hist-table" },
    ["button", { class: "scroll-up-button" }, "Scroll Up"],
  ],
});

export default ScrollUp;
